napari_plugin_manager/qt_plugin_dialog.py

```diff
@@ -965,6 +969,9 @@ def setup_ui(self):
 
         self.packages_filter.setFocus()
 
+        stylesheet = get_current_stylesheet([STYLES_PATH])
+        self.setStyleSheet(stylesheet)
```

Comment:
Why is this need twice? It's a bit weird, but if that's how it is, so be it 🤷 Let's write a comment at least.

Reply:
Indeed que can only use the one in setupui, removing the one on init